targets/DeployDependencies.proj

```diff
@@ -12,10 +12,17 @@
 
       <!-- For each package, get its content files. -->
       <ContentFilesFromPackage Include="%(PackagesFoldersInPackagesDir.Identity)content\**\*.*" />
-      <DestinationFilesFromPackage Include="@(ContentFilesFromPackage->'$(OutputPath)\%(RecursiveDir)%(Filename)%(Extension)')" />
     </ItemGroup>
+
+    <!-- Copy the dependencies to both the deployment and the test directories -->
+    <ItemGroup>
+      <DeploymentDestination Include="@(ContentFilesFromPackage->'$(DeploymentDir)\%(RecursiveDir)%(Filename)%(Extension)')" />
+      <DeploymentDestination Include="@(ContentFilesFromPackage->'$(TestDeploymentDir)\%(RecursiveDir)%(Filename)%(Extension)')" />
+    </ItemGroup>
+
+    <!-- Copy the package contents to the deployment directory -->
     <Copy SourceFiles="@(ContentFilesFromPackage)"
-          DestinationFiles="@(DestinationFilesFromPackage)"
+          DestinationFiles="%(DeploymentDestination.Identity)"
           SkipUnchangedFiles="true"
```

Comment:
ContentFilesFromPackage does not have one-to-one correspondence to DeploymentDestination. I would just do to Copy tasks.